contrib/conflict_resolution/resolve_interactively.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""Ask user to resolve a vdirsyncer sync conflict interactively.
+
+Needs a way to ask the user.
+The use of https://apps.kde.org/kdialog/ for GNU/Linix is hardcoded.
+
+Depends on python>3.5 and KDialog.
+
+Usage:
+  Ensure the file executable and use it in the vdirsyncer.conf file, e.g.
+
+  conflict_resolution = ["command", "/home/bern/vdirsyncer/resolve_interactively.py"]
+
+This file is Free Software under the following license:
+SPDX-License-Identifier: Apache-2.0
+SPDX-FileCopyrightText: 2021 Intevation GmbH <https://intevation.de>
+Author: <bernhard.reiter@intevation.de>
+"""
+from pathlib import Path
+import re
+import subprocess
+import sys
+
+KDIALOG = "/usr/bin/kdialog"
```

Comment:
The hardcode is fine; I'm sure someone else interested in using the script will come along and make it parametrizeable.

Reply:
Note that the interface to kdialog is implemented in the whole script, like parameter names and return codes, but it serves as an example how to do it, even with a different interactive application.